code/modules/surgery/organs/augments_chest.dm

```diff
@@ -218,3 +218,127 @@
 			owner.adjustFireLoss(10)
 			to_chat(owner, span_danger("Your thruster implant malfunctions and mildly burns you!"))
 
+/obj/item/organ/cyberimp/chest/spinalspeed
+	name = "neural overclocker implant"
+	desc = "Overloads your central nervous system in order to do everything faster. Careful not to overuse it."
+	slot = ORGAN_SLOT_THRUSTERS
+	icon_state = "imp_spinal"
+	implant_overlay = null
+	implant_color = null
+	actions_types = list(/datum/action/item_action/organ_action/toggle)
+	w_class = WEIGHT_CLASS_NORMAL
+	syndicate_implant = TRUE
+	var/on = FALSE
+	var/time_on = 0
+	var/warningcooldown = 0
```

Comment:
use cooldown defines https://github.com/yogstation13/Yogstation/blob/master/code/__DEFINES/cooldowns.dm#L64-L73

Reply:
got it, was just heading to bed, but i'll fix this tomorrow